app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -377,14 +360,17 @@ public boolean onOptionsItemSelected(final MenuItem item) {
                 return true;
             }
             case R.id.action_preview: {
+                saveDocumentPositions();
                 setDocumentViewVisibility(true);
```

Comment:
Does it make more sense to call `saveDocumentPositions` inside `setDocumentViewVisibility`

Reply:
might be that setting cursor happens after saving (=->always 0), caution

Reply:
> 
> 
> might be that setting cursor happens after saving (=->always 0), caution

yes, it is a reason that I do not move `saveDocumentPositions` into `setDocumentVisibility`